src/index.js

```diff
@@ -18,6 +18,7 @@ class FtpServer extends EventEmitter {
       pasv_min: 1024,
       pasv_max: 65535,
       pasv_url: null,
+      public_pasv_url: null,
```

Comment:
I don't see the need for this. The `pasv_url` should be your public IP that is provided to the client.

Reply:
I tried to set that but then it was not able to find passive ports using the public IP

Reply:
Sounds like [this](https://github.com/trs/ftp-srv/pull/155)? If so, that'll be fixed in [v4.0.1](https://github.com/trs/ftp-srv/releases/tag/v4.0.1)

Reply:
you are correct